### PR TITLE
Ported barks from splurt

### PR DESCRIPTION
**Pull request by prinnygod**
## About The Pull Request

Added Bark sounds ported from splurt. I'll add some more, but this is more of a practice run, never did anything like this but this should work fine as I tested it out in game. 

Banjo kazooie sounds from: https://github.com/SPLURT-Station/S.P.L.U.R.T-Station-13/pull/638
other sounds: https://github.com/SPLURT-Station/S.P.L.U.R.T-Station-13/pull/379

## Why It's Good For The Game

More stuff 

## Changelog ##

🆑 
add: New bark sounds ported from splurt
/:cl:

### CODE REVIEW

**Comment by nednaZ** [comment]
Friend of theirs helping with their PR.

Make sure the About and "Why it's good for the game" sections are filled in, no text should be above those headers.
For the changelog there, don't make it "added" as it won't get picked up by the auto-changelog.
Readd the 🆑 and /:cl:  tags, with "add:" there instead.

Those tags are automatically read by a bot and made into real changelog.

File: code/datums/bark.dm [hunk 1]
```diff
@@ -96,6 +96,95 @@
...
+
+/datum/bark/speak4
+	name = "speak_4"
+	id = "speak_4"
+	soundpath = 'sound/voice/barks/speak_4.ogg'
+
+/datum/bark/zap
+	name = "zap"
+	id = "zap"
+	soundpath = 'sound/voice/barks/zap.ogg'
```
Comment (nednaZ) [inline code/datums/bark.dm:187]: I'm unsure how this server handles modularization, but you can possibly move these to (for example), lumos/code/datums/bark.dm to separate the changes out for ease of upstream pulls.

The sounds themselves can also go into lumos/sound/voice/barks/???.ogg

**Comment by nednaZ** [comment]
Make sure to credit, if possible, the original pull requests that you're porting from by leaving links in the main description itself.

**Comment by iknowdude00** [comment]
Modular folder has been added to the repo. Please adjust PR accordingly, creating correct folder paths etc. Just the same path that the core bark sounds use but under modular_lumos

/modular_lumos/code/datums/bark.dm
/modular_lumos/sound/voice/barks/

etc.

File: code/datums/bark.dm [hunk 2]
```diff
@@ -95,8 +95,6 @@
 	name = "Brassy"
 	id = "coggers"
 	soundpath = 'sound/machines/clockcult/integration_cog_install.ogg' //Yet another unexpectedly good bark sound
-
-
```
Comment (iknowdude00) [inline code/datums/bark.dm:99]: Deleted a couple too many lines there.

File: modular_lumos/code/datums/bark.dm [hunk 3]
```diff
@@ -0,0 +1,93 @@
+//Datums for barks and bark accessories
+
+/datum/bark
```
Comment (nednaZ) [inline modular_lumos/code/datums/bark.dm:3]: Unneeded double definition, just delete the /datum/bark

**Review by nednaZ — COMMENTED** [review]
Helping a friend again.

**Pull request merged**